components/02-form-elements/currency/currency.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="input-type input-type--currency">
+<div class="input-type input-type--prefix">
+  <input class="input input-type__input" name="answer" id="answer" type="number" data-qa="input-text">
   <abbr title="{{label}}"  class="input-type__type" id="answer-type">{{currency}}</abbr>
```

Comment:
@bameyrick  Issue with the alignment of currency fields
<img width="406" alt="screen shot 2018-11-12 at 16 01 51" src="https://user-images.githubusercontent.com/43346934/48359596-32b46980-e695-11e8-86a7-488abbe32daf.png">

Reply:
![screenshot 2018-11-13 at 11 37 12](https://user-images.githubusercontent.com/8447446/48411110-7f04b580-e738-11e8-8d95-bd6216471d4e.png)

Reply:
@jrbarnes9 This should be fixed now